docs/anexia.md

```diff
@@ -6,4 +6,14 @@ This provider implementation is currently in **alpha** state.
 
 Only flatcar linux is currently supported and you explicitly have to set the provisioning mechanism to cloud-init by setting `machine.spec.providerSpec.value.operatingSystemSpec.provisioningUtility` to "cloud-init".
 
-An example machine deployment can be found here: [examples/anexia-machinedeployment.yaml](../examples/anexia-machinedeployment.yaml)
\ No newline at end of file
+An example machine deployment can be found here: [examples/anexia-machinedeployment.yaml](../examples/anexia-machinedeployment.yaml)
+
+## Templates
+
+To retrieve all available templates against a given location:
+
+```
+https://engine.anexia-it.com/api/vsphere/v1/provisioning/templates.json/<location identifier>/templates?page=1&limit=50
```

Comment:
Just curious, does this work without any credentials? I don't know any location identifiers so no idea how to try it out. Maybe we can supplement a CLI call that prints available templates?

Reply:
You need the authentication token for this https://engine.anexia-it.com/api/vsphere/doc/

Reply:
This is not supposed to be an unauthenticated/public endpoint.

Reply:
Ah, gotcha. Can we include that information in the doc maybe?

Reply:
I don't think it's required but sure :)